index.html

```diff
@@ -173,6 +173,31 @@ <h2>About <br> Myself</h2>
                     <li class="list-item"><a href="#">Skills</a> <i class="fa fa-angle-right"></i></li>
                 </ul>
             </section>
+            <section class="contact_form">
+              <div class="contact-div">
+                <h2 class="contact-h2">Contact me</h2>
+                <p class="contact-paragraph">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+                </p>
+                <form action="/contact-me" method="post">
+                    <ul class="contact">
+                        <li class="form-contact">
+                            <input type="text" name="name" placeholder="Type Your Name">
+                        </li>
+                        <li class="form-contact">
+                            <input type="email" name="email" placeholder="Type your email">
+                        </li>
+                        <li class="contact-form">
+                            <textarea name="message" cols="35" rows="10">Type your message</textarea>
+                        </li>
+                        <li class="contact-form">
+                            <button type="submit" class="btn">Get it touch</button>
+                        </li>
+                    </ul>
```

Comment:
- [ ] You can use simple form structure, without the `ul`/`li`. I know, you can fix it easily 👍